### PR TITLE
Allow code class as string, e.g. "1xx" or "informal"

### PR DESCRIPTION
**Pull request by afh**
so folks can do:

```
% hssp class 1
% hssp class 1xx
% hssp class informal # case does not matter, e.g. Informal, InFoRmAL, etc. are accepted
```

### CODE REVIEW

File: internal/status/status.go [hunk 1]
```diff
@@ -60,6 +61,33 @@ func (s Status) GiveClassName() string {
...
+func CodeClassFromName(name string) (int, bool) {
+	switch strings.ToLower(name) {
+	case "1xx":
+		fallthrough
+	case "informal":
+		return Informal, true
+	case "2xx":
+		fallthrough
+	case "successfull":
+		return Successfull, true
```
Comment (afh) [inline internal/status/status.go:73]: ℹ️ Change as follows once #7 or #4 are merged:

```suggestion
	case "successful":
		return Successful, true
```

File: internal/status/status.go [hunk 2]
```diff
@@ -60,6 +61,33 @@ func (s Status) GiveClassName() string {
 	return "Unassigned"
 }
 
+func CodeClassFromName(name string) (int, bool) {
+	switch strings.ToLower(name) {
+	case "1xx":
+		fallthrough
+	case "informal":
+		return Informal, true
```
Comment (sterchelen) [inline internal/status/status.go:69]: I would prefer to combine each class in its own case, like:

```go
case "1xx", "informal":
  return Informal, true
``` 

wdyt?

Comment (afh) [reply internal/status/status.go:69]: Great suggestion, @sterchelen! Apologies, my Go has been a bit rusty these days…

**Review by sterchelen — APPROVED** [review]
Nice, thx again.

**Comment by afh** [comment]
Thanks for reviewing and accepting this PR, @sterchelen, much appreciated :)

**Pull request merged**